decidim-proposals/config/locales/en.yml

```diff
@@ -70,9 +81,25 @@ en:
           fields:
             category: Category
             official_proposal: Official proposal
+            reports: Reports
             scope: Scope
             state: State
             title: Title
+      not_found: The proposal doesn't exists.
+      proposal_reported_mailer:
+        hide:
+          hello: Hello %{name},
+          report_html: The proposal %{proposal_link} has been hidden automatically.
+          subject: A proposal has been hidden automatically
+        report:
+          hello: Hello %{name},
+          report_html: The proposal %{proposal_link} has been reported by %{reporter_name}
```

Comment:
I also find this quite intrusive, the fact we store the user is to prevent duplicates, not actually render it. Not rendering the name looks like the way to go.